src/router.js

```diff
@@ -208,19 +217,9 @@ export class Router {
 
     config.navModel = navModel;
 
-    if ((config.nav || 'order' in navModel) && this.navigation.indexOf(navModel) === -1) {
-      navModel.order = navModel.order || config.nav;
-      navModel.href = navModel.href || config.href;
-      navModel.isActive = false;
-      navModel.config = config;
```

Comment:
Is that just due to a refactoring? I want to make sure that all of this is still available as part of the new nav model. Devs need it to bind to in their UI. Also, settings needs to be present since that's considered a "safe place" for developer to put a custom object or data.